watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Users/UserHeaderAdapter.java

```diff
@@ -0,0 +1,89 @@
+package com.blueprint.watershed.Users;
+
+import android.app.Activity;
+import android.graphics.Typeface;
+import android.support.v4.app.Fragment;
+import android.util.TypedValue;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import com.blueprint.watershed.R;
+import com.blueprint.watershed.Tasks.TaskAbstractFragment;
+
+import java.util.List;
+
+/**
+ * Created by Mark Miyashita on 11/19/14.
+ */
+public class UserHeaderAdapter extends ArrayAdapter<User> {
+
+    Activity mMainActivity;
+    List<User> mUsers;
+    Fragment mFragment;
+
+    String HEADER = "header";
+    String ROW = "row";
+
+    public UserHeaderAdapter(Activity mainActivity, List<User> users, Fragment fragment) {
+        super(mainActivity, R.layout.user_list_row, users);
+        mMainActivity = mainActivity;
+        mUsers = users;
+        mFragment = fragment;
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+        View row = convertView;
+        UserHolder holder;
+        final User user = getItem(position);
+
+        if (row == null) {
+            LayoutInflater inflater = mMainActivity.getLayoutInflater();
+            row = inflater.inflate(R.layout.user_list_row, parent, false);
+
+            holder = new UserHolder();
+            holder.mTextView = (TextView) row.findViewById(R.id.user_row_text);
+            holder.mLine = row.findViewById(R.id.user_row_color);
+
+            row.setTag(holder);
+        } else {
+            holder = (UserHolder) row.getTag();
+        }
+
+        String text;
+        int fontSize;
+        int fontWeight;
+        if (user.getLayoutType() == null) {
```

Comment:
So is the LayoutType just a flag for whether or not it's a header? In that case, we should just set an actual boolean flag. This seems kind of hacky otherwise.